Patch to find casacore and casa files in new /opt location
Index: carta-backend-1.4.1+fixed1/CMakeLists.txt
===================================================================
--- carta-backend-1.4.1+fixed1.orig/CMakeLists.txt
+++ carta-backend-1.4.1+fixed1/CMakeLists.txt
@@ -3,6 +3,7 @@ project(carta_backend)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_INCLUDE_DIRECTORIES_BEFORE ON)
+set(CMAKE_LINK_DIRECTORIES_BEFORE ON)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/install)
 include(uWebSockets)
@@ -44,8 +45,8 @@ INCLUDE_DIRECTORIES(${LIBXML2_INCLUDE_DI
 ADD_SUBDIRECTORY(carta-protobuf)
 ADD_SUBDIRECTORY(carta-scripting-grpc)
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
-INCLUDE_DIRECTORIES(/usr/include/casacore /usr/include/casacode)
-INCLUDE_DIRECTORIES(/usr/local/include/casacore /usr/local/include/casacode)
+INCLUDE_DIRECTORIES(/opt/carta-casacore/include /opt/carta-casacore/include/casacore /opt/carta-casacore/include/casacode)
+LINK_DIRECTORIES(/opt/carta-casacore/lib)
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     INCLUDE_DIRECTORIES(/usr/local/include)
